app/shop.py

```diff
@@ -0,0 +1,32 @@
+class Shop:
+
+    def __init__(self, name, location: list, products: dict):
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def full_amount_shopping(self, customer, petrol):
+        coast = 0
+
+        for key, value in self.products.items():
+            coast += value * customer.product_cart[key]
+
+        full_coast = round(customer.distance(
+            self.location) * 2 / 100 * customer.car[
+            "fuel_consumption"] * petrol + coast, 2)
+        return full_coast
+
+    def check_shop(self, customer):
+        """check - complete information about each purchase"""
+        check = ""
```

Comment:
```suggestion
        bill = ""
```
Якщо мається на увазі чек як в магазині, то краще використовувати слово "bill"